docs/blueprints/EP036.rst

```diff
@@ -111,13 +141,26 @@ Events
 
 Subscribed
 -----------
+
   - **kytos/flow_manager.flow.added**
   - **kytos/of_core.handshake.completed**
+  - **kytos/flow_manager.flow.error**
+  - **kytos/[mef_eline|telemetry_int|coloring|of_lldp].enable_table** - Every NApp sends an event with their table settings as content.
+
+kytos/<napp_name>.enable_table
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This event is sent from other NApps with information about their table settings. It is empty in case of failure.

```

Comment:
That'd work, my concern with the event content modeling is that there's no clear way to know the reason of the error. It would be great for `multi_table` to have a way to get the error reason too. That way we could also expose a `status_reason: list[str]` to also expose this in the API, wdyt? We discussed that a pipeline shouldn't start the deployment once the error happens but although we have async events we still need to handle/bubble up errors.

Reply:
Something like:
```
  "content": {
    "error": <reason>
  }
```
could work but how does the data for reason look like? Is there a specific type for them?

Reply:
A `str` should suffice since we only need the content to bubble up without having to identify which type of error, and treat all errors as unrecoverable. However, based on how the conversation unfloded here https://github.com/kytos-ng/kytos/pull/359#discussion_r1160892793, I believe now that in this iteration we won't need to handle errors from `kytos/<napp_name>.enable_table` right?

Reply:
Maybe for when a NApp does not find its name in the event from `kytos/multi_table.enable_table`. Or maybe a `warning` is enough in this case since it is not critical.

Reply:
Let's leave at is is, `kytos/<napp_name>.enable_table` is already targeting each NApp as opposed to by being a general event topic, so if an accidental error happens let's let it crash since it's really not expected in this part and if it ever happens it should be fixed. OK?

In the new event that each NApp will publish their cookie prefix https://github.com/kytos-ng/kytos/pull/359#discussion_r1160883967, let's also add which table group they support, just so `multi_table` can keep track of this and validate at request level all known table groups, OK? 

Other than that we can close this thread here, thanks.